src/Fantomas/Context.fs

```diff
@@ -416,10 +416,10 @@ let internal sepSpace (ctx : Context) =
 
 let internal sepNln = !+ ""
 
-let internal whenLastEventIsNotWriteLine f (ctx: Context) =
+let internal sepNlnUnlessLastEventIsNewline (ctx: Context) =
```

Comment:
all usages of this where with sepNln passed to it so I just renamed this function and updated the usages, let me know if you're ok with that

Reply:
Ok, with renaming. Makes sense.